### PR TITLE
react-input: write readme

### PR DESCRIPTION
**Pull request by spmonahan**
## Current Behavior

`react-input`'s README was incomplete.

## New Behavior

`react-input`'s README now:

1. Provides a short summary of what the component is for
2. Shows sow to import the component
3. Show basic usage 
4. Provides links to relevant documentation

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                                                                                             |      Baseline (minified/GZIP) |                           PR |                                                                                                Change |
| :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: | ---------------------------: | ----------------------------------------------------------------------------------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `179.813 kB`<br />`50.186 kB` | `179.813 kB`<br />`50.19 kB` | <br />`4 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                  |         Size (minified/GZIP) |
| -------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr> | `34.178 kB`<br />`11.138 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                                            |  `22.274 kB`<br />`7.336 kB` |
| <samp>react-spinbutton</samp> <br /> <abbr title='bundle-size/SpinButton.fixture.js'>SpinButton</abbr>                                             | `41.985 kB`<br />`11.908 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/f120fe9160ad7311a87af0cfacd8d2fb26275e34'>f120fe9160ad7311a87af0cfacd8d2fb26275e34</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/FluentProviderWithTheme-mount_base.html">345</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/FluentProviderWithTheme-mount.html">345</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/FluentProviderWithTheme-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">272</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">272</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">371</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">331</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/Avatar-mount_base.html">1048</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/Avatar-mount.html">1102</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/Button-mount_base.html">676</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/Button-mount.html">623</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/FluentProvider-mount_base.html">2186</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/FluentProvider-mount.html">2223</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/FluentProviderWithTheme-mount_base.html">345</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/FluentProviderWithTheme-mount.html">345</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/FluentProviderWithTheme-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">272</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">272</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">371</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">331</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/MakeStyles-mount_base.html">1808</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22993/perf-test-react-components/MakeStyles-mount.html">1858</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: packages/react-components/react-input/README.md [hunk 1]
```diff
@@ -3,3 +3,35 @@
...
 These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
+
+Inputs give people a way to enter and edit text.
+
+### Usage
+
+Import Input:
+
+```js
+import { Input } from '@fluentui/react-input';
```
Comment (behowell) [inline packages/react-components/react-input/README.md:14]: Should this reference react-components instead?
```suggestion
import { Input } from '@fluentui/react-components';
```

File: packages/react-components/react-input/README.md [hunk 2]
```diff
@@ -3,3 +3,35 @@
...
+```
+
+#### Examples
+
+```jsx
+<Input defaultValue="Hello, World!"/>
+<Input value={value} onChange={onInputChange}/>
```
Comment (behowell) [inline packages/react-components/react-input/README.md:21]: nit: space before `/>`
```suggestion
<Input defaultValue="Hello, World!" />
<Input value={value} onChange={onInputChange} />
```

**Review by behowell — APPROVED** [review]

File: packages/react-components/react-input/README.md (hunk 1 above)
Comment (spmonahan) [reply packages/react-components/react-input/README.md:14]: I'm going to make that change once I move the component to RC and it's available at that path.

**Pull request merged**